
	Python anaconda is now loaded in your environment.

Preparing directory /jmain01/home/JAD017/sjr02/dxg49-sjr02/kfac-curvature/out/ImageNet32/PreResNet110/SWA/lr=0.1_wd=8e-06_swastart=176.0_swalr=0.01/
Using model PreResNet110
Loading ImageNet32 from /jmain01/home/JAD017/sjr02/dxg49-sjr02/curvature/data/
You are going to run models on the test set. Are you sure?
Using train (1281167) + test (50000)
Preparing model
{'depth': 110}
SWAG training
Traceback (most recent call last):
  File "../run_swag.py", line 136, in <module>
    time_ep = time.time() - time_ep
NameError: name 'time_ep' is not defined
Preparing directory /jmain01/home/JAD017/sjr02/dxg49-sjr02/kfac-curvature/out/ImageNet32/PreResNet110/SWA/lr=0.1_wd=8e-06_swastart=176.0_swalr=0.05/
Using model PreResNet110
Loading ImageNet32 from /jmain01/home/JAD017/sjr02/dxg49-sjr02/curvature/data/
You are going to run models on the test set. Are you sure?
Using train (1281167) + test (50000)
Preparing model
{'depth': 110}
SWAG training
Resume training from /jmain01/home/JAD017/sjr02/dxg49-sjr02/kfac-curvature/out/ImageNet32/PreResNet110/SWA/lr=0.1_wd=8e-6_swastart=176_swalr=0.01/checkpoint-00175.pt
Traceback (most recent call last):
  File "../run_swag.py", line 117, in <module>
    checkpoint = torch.load(args.resume)
  File "/jmain01/home/JAD017/sjr02/dxg49-sjr02/.conda/envs/diegorubin/lib/python3.7/site-packages/torch/serialization.py", line 419, in load
    f = open(f, 'rb')
FileNotFoundError: [Errno 2] No such file or directory: '/jmain01/home/JAD017/sjr02/dxg49-sjr02/kfac-curvature/out/ImageNet32/PreResNet110/SWA/lr=0.1_wd=8e-6_swastart=176_swalr=0.01/checkpoint-00175.pt'
Preparing directory /jmain01/home/JAD017/sjr02/dxg49-sjr02/kfac-curvature/out/ImageNet32/PreResNet110/SWA/lr=0.1_wd=0.0_swastart=176.0_swalr=0.01/
Using model PreResNet110
Loading ImageNet32 from /jmain01/home/JAD017/sjr02/dxg49-sjr02/curvature/data/
You are going to run models on the test set. Are you sure?
Using train (1281167) + test (50000)
Preparing model
{'depth': 110}
SWAG training
Resume training from /jmain01/home/JAD017/sjr02/dxg49-sjr02/kfac-curvature/out/ImageNet32/PreResNet110/SWA/lr=0.1_wd=8e-6_swastart=176_swalr=0.01/checkpoint-00175.pt
Traceback (most recent call last):
  File "../run_swag.py", line 117, in <module>
    checkpoint = torch.load(args.resume)
  File "/jmain01/home/JAD017/sjr02/dxg49-sjr02/.conda/envs/diegorubin/lib/python3.7/site-packages/torch/serialization.py", line 419, in load
    f = open(f, 'rb')
FileNotFoundError: [Errno 2] No such file or directory: '/jmain01/home/JAD017/sjr02/dxg49-sjr02/kfac-curvature/out/ImageNet32/PreResNet110/SWA/lr=0.1_wd=8e-6_swastart=176_swalr=0.01/checkpoint-00175.pt'
